### ISSUE TITLE
toBuilder-Method and primitive type boolean

### ISSUE THREAD

**Issue by radtke** [skinny85/jilt#18]
Hi there,

i think maybe there's a bug generating the "toBuilder"-Method for Classes containing primitive type booleans.
The problem is in `AbstractBuilderGenerator.accessAttributeOfTargetClass(..)` which does not consider standard naming convention for primitive type boolean getter ("is" instead of "get").

Here is an example.

My Class:

```java
@Builder(toBuilder = "from")
public class Subscription {
  private String label;
  private long number;
  private boolean active;

  public Subscription(String label, long number, boolean active) {
    super();
    this.label = label;
    this.number = number;
    this.active = active;
  }

  public String getLabel() {
    return label;
  }

  public void setLabel(String label) {
    this.label = label;
  }

  public long getNumber() {
    return number;
  }

  public void setNumber(long number) {
    this.number = number;
  }

  public boolean isActive() {
    return active;
  }

  public void setActive(boolean active) {
    this.active = active;
  }
}
```

Generated Builder Class:

```java
@Generated("Jilt-1.5")
public class SubscriptionBuilder {
  private String label;

  private long number;

  private boolean active;

  public static SubscriptionBuilder subscription() {
    return new SubscriptionBuilder();
  }

  public static SubscriptionBuilder from(Subscription subscription) {
    SubscriptionBuilder subscriptionBuilder = new SubscriptionBuilder();
    subscriptionBuilder.label(subscription.getLabel());
    subscriptionBuilder.number(subscription.getNumber());
    subscriptionBuilder.active(subscription.active);
    return subscriptionBuilder;
  }

  public SubscriptionBuilder label(String label) {
    this.label = label;
    return this;
  }

  public SubscriptionBuilder number(long number) {
    this.number = number;
    return this;
  }

  public SubscriptionBuilder active(boolean active) {
    this.active = active;
    return this;
  }

  public Subscription build() {
    return new Subscription(label, number, active);
  }
}
```

And of course, `subscriptionBuilder.active(subscription.active);` does not compile.

**Comment by skinny85** [comment]
Hey @radtke,

thanks for opening the issue! That's a great catch, yes, the logic of figuring out the accessor should definitely also try the "is<PropertyName>" variant as well.

I'll work on a fix soon.

Thanks,
Adam

**Comment by radtke** [comment]
Thanks for your quick reply and your effort!

**Comment by skinny85** [comment]
Ok, this has been now fixed on `develop` (https://github.com/skinny85/jilt/commit/027dbd5f5e44f1bda2fb23a2ff04fd2dedf25fc6), and will be released with the next version of Jilt (`1.6`).

**Comment by skinny85** [comment]
This bug has been fixed in Jilt version `1.6`.

Let me know if you run into any problems related to this, and I'd be happy to reopen this issue.
